mount/mount_windows.go

```diff
@@ -78,13 +138,51 @@ func (m *Mount) GetParentPaths() ([]string, error) {
 
 // Unmount the mount at the provided path
 func Unmount(mount string, flags int) error {
+	fi, err := os.Lstat(mount)
+	if err != nil {
+		return errors.Wrapf(err, "unable to find mounted volume %s", mount)
+	}
+
+	// Remove any trailing slashes
+	mount = filepath.Clean(mount)
+	dir, file := filepath.Split(mount)
```

Comment:
looks like this is not necessary, dir and file are only used on the next line `:layerid` could be appended directly to the call to `ReadFile`